public/src/components/channelManagement/createTestDialog.tsx

```diff
@@ -90,6 +119,33 @@ const CreateTestDialog: React.FC<CreateTestDialogProps> = ({
         </IconButton>
       </div>
       <DialogContent dividers>
+        {testNamePrefix === undefined && (
```

Comment:
if `testNamePrefix` is defined then we don't allow a campaign (for 'holdback' epic only)